libr/core/cmd_anal.c

```diff
@@ -9931,6 +9931,7 @@ static void cmd_anal_classes(RCore *core, const char *input) {
 		}
 		RAGraph *agraph = create_agraph_from_graph (graph);
 		if (!agraph) {
+			r_graph_free (graph);
```

Comment:
this is the only "fix" in this PR, right? btw see the annotations below, but i think its not related to your pr

Reply:
Not, there is one more, there was missing return in class.c

Reply:
I forgot to push one more missing free from my laptop in rtti_itanium.c, it's there now